src/backend/functions/orchestrator/lib/index.js

```diff
@@ -102,35 +98,103 @@ function createDeleteParams({mediaUrl, tasksTableName}) {
    };
 }
 
-// terminated :: ({k:v} -> Promise {k: v}) -> {k: v} -> String -> Promise {k:v}
+// terminated :: ({k:v} -> Promise {k: v}) -> {k: v} -> {k: v} -> Promise {k:v}
 const terminated = R.curry(deleteItem =>
    handle(o(deleteItem, createDeleteParams), ({MediaUrl: mediaUrl}) => ({mediaUrl}))
 );
 
-function error(ecs, ddb) {}
+const errorUpdate = R.curry((updateItem, {tasksTableName, mediaUrl, taskArn}) => {
+   const params = {
+      TableName: tasksTableName,
+      Key: {
+         MediaUrl: mediaUrl
+      },
+      UpdateExpression: 'ADD Retries :val SET TaskStatus = :status, TaskArn = :task',
+      ExpressionAttributeValues: {
+         ':val': 1,
+         ':status':  'WAITING',
```

Comment:
one for a future PR on docs - would be good to have a list of statuses and their meaning

Reply:
I think I'm wrong here tbh, but if we update the item status to `WAITING` and `startTranscription` as part of the error handler, and the waiting handler also calls `startTranscription`, won't we have 2 on the go at once?

Reply:
No, because DynamoDb streams guarantee temporal ordering for individual DynamoDb item so an item that is changed in this invocation will have to wait until the next invocation before that change is processed.